std_msgs/Header header
ouroboros_ros/VlcImageMetadataMsg metadata
ouroboros_sor/SparkImageMsg image
sensor_msgs/Image embedding
sensor_msgs/Image keypoints
sensor_msgs/Image descriptors
geometry_msgs/PoseStamped pose_hint


